src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1191,6 +1204,19 @@ def _private_key_bytes(self, encoding, format, encryption_algorithm,
         assert res == 1
         return self._read_mem_bio(bio)
 
+    def _private_key_bytes_traditional_der(self, type, cdata):
+        if type == self._lib.EVP_PKEY_RSA:
+            write_bio = self._lib.i2d_RSAPrivateKey_bio
+        else:
+            raise TypeError(
+                "Only RSA keys are supported for DER serialization"
+            )
+
+        bio = self._create_mem_bio()
+        res = write_bio(bio, cdata)
```

Comment:
I assume the `write_bio` indrection here is because there'll be more branches on that `if` shortly?